src/test/java/com/epam/reportportal/listeners/ListenerUtilsTest.java

```diff
@@ -34,4 +40,23 @@ public void getLaunchMode() {
 		Assert.assertEquals(DEFAULT, ListenersUtils.getLaunchMode("notvalid"));
 		Assert.assertEquals(DEBUG, ListenersUtils.getLaunchMode("Debug"));
 	}
+
+	@Test
+	public void handleException_restEndpointIOExceptionAndNullLogger() {
```

Comment:
Do we really need this tests? What if logger is configured for files instead for system console?